ansible/centos-ci/ansible/roles/vagrant/tasks/main.yml

```diff
@@ -0,0 +1,35 @@
+# file: ansible/roles/vagrant/install.yml
+---
+
+# For some reason this task gets always skipped :/
+#- name: "Install Vagrant from SCL"
+#  include_role:
+#    name: smbambling.scl
```

Comment:
Yup, installed it via Ansible galaxy, otherwise Ansible complains that the given role is missing and bails. (I could actually see the the first step of that role being executed, which checks for the presence of SCL GPG key).